core/server/models/post.js

```diff
@@ -40,12 +41,18 @@ Post = ghostBookshelf.Model.extend({
      *      2. model events e.g. "post.published" are using the inserted resource, not the fetched resource
      */
     defaults: function defaults() {
+        let visibility = 'public';
+
+        if (settingsCache.get('labs') && (settingsCache.get('labs').members === true) && settingsCache.get('labs').default_content_visibility) {
```

Comment:
@allouis have placed the `default_content_visibility` in labs next to existing `members` flag but later have seen `members_subscription_settings` and other's being put directly into settingsCache. What do you think? Does the flag fit here or should it be moved out to cache and used in similar manner like `members_subscription_settings`?